apps/state/lib/state/stop.ex

```diff
@@ -86,6 +86,10 @@ defmodule State.Stop do
     State.Stop.Cache.match(%{location_type: type, parent_station: id}, :parent_station)
   end
 
+  def by_vehicle_type(vehicle_types) do
```

Comment:
nitpick: since this function takes a list of vehicle types
```suggestion
  def by_vehicle_types(vehicle_types) do
```